lib/seattleflu/db/cli/command/etl/longitudinal.py

```diff
@@ -374,7 +375,30 @@ def symptoms(document: dict) -> list:
     parent_reported_symptoms = document.get("sx_specific") or []
     rn_reported_symptoms = document.get("rn_sx") or []
 
-    return list(set(parent_reported_symptoms + rn_reported_symptoms))
+    symptoms = list(set(parent_reported_symptoms + rn_reported_symptoms))
+
+    symptoms_map = {
+        "ear_pain": "earPainOrDischarge",
+        "fever": "feelingFeverish",
+        "headache": "headaches",
+        "myalgia": "muscleOrBodyAches",
+        "ndv": ["nauseaOrVomiting", "diarrhea"],
+        "nvd": ["nauseaOrVomiting", "diarrhea"],
+        "runny_nose": "runnyOrStuffyNose",
+        "sore_throat": "soreThroat",
+        "wob": "increasedTroubleBreathing",
+        "cough": "cough",
+        "fatigue": "fatigue",
+        "rash": "rash",
+    }
```

Comment:
I'm curious to know if either of you have a better idea for how to handle "cough" = "cough" -type mapping. I wanted to avoid a db lookup of existing symptoms, and the only other idea I had was to check inside a list `["rash", "fatigue", "cough"]` before mapping. I didn't like that idea very much, though.

Reply:
I will update the `standardize_symptom` function depending on what direction we want to go with this.

Reply:
I think it's fine to have an explicit identity entry (a = a) in the map.

I have some other notes on this which I pushed as a fixup commit.